src/Layers/xrRender/light.cpp

```diff
@@ -208,11 +208,12 @@ void light::spatial_move()
     // update spatial DB
     SpatialBase::spatial_move();
 
-#if (RENDER == R_R2) || (RENDER == R_R3) || (RENDER == R_R4)
-    if (flags.bActive)
+#if (RENDER == R_GL) || (RENDER == R_R2) || (RENDER == R_R3) || (RENDER == R_R4)
+    if (flags.bActive) {
```

Comment:
Please, fix the code style: brace should always be on a new line.
```suggestion
    if (flags.bActive)
    {
```

Reply:
@Xottab-DUTY fix here https://github.com/OpenXRay/xray-16/pull/1148/commits/a6e64361ad518397516a76a5f9bd99481aa3c879

Reply:
Don't forget about other braces in this pull request – they should be fixed too ;)

Reply:
So...?